From 3e3f5c37252a33ea1e71c39f2ca0f13940c261ad Mon Sep 17 00:00:00 2001
From: Martin Weinelt <hexa@darmstadt.ccc.de>
Date: Sat, 17 Jul 2021 16:11:23 +0200
Subject: [PATCH] tests: ignore OSErrors in hass fixture

The nix sandbox will cause OSErrors due to limitations imposed on
network interaction. This change makes it so we forgive these cases.
---
 tests/conftest.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/tests/conftest.py b/tests/conftest.py
index 1f5ffc80d0..b284727a0f 100644
--- a/tests/conftest.py
+++ b/tests/conftest.py
@@ -168,6 +168,8 @@ def hass(loop, load_registries, hass_storage, request):
             continue
         if isinstance(ex, ServiceNotFound):
             continue
+        if isinstance(ex, OSError):
+            continue
         raise ex
 
 
-- 
2.32.0

